src/coredump/coredump.c

```diff
@@ -775,6 +779,10 @@ static int submit_coredump(
         if (r < 0)
                 return log_error_errno(r, "Failed to log coredump: %m");
 
+        emtr_event_recorder_record_event_sync (emtr_event_recorder_get_default (),
+                                               PROGRAM_DUMPED_CORE_EVENT,
+                                               NULL);
```

Comment:
Uhh... could we log something more useful here such as... what the program was? Maybe the version of the program (checksum) or ostree so we can relate it back to things starting and stopping crashing?

Reply:
Wow, brainfart moment on my end. To be honest, I think @cosimoc was correct about this on the ticket, we don't want to do this in systemd anyway - it'd be far better to have our own crash reporter linked to eosmetrics.